eng/pipelines/templates/jobs/ci.yml

```diff
@@ -44,17 +26,24 @@ jobs:
     TestPipeline: ${{ parameters.TestPipeline }}
     PipelineArtifactName: ${{ parameters.PipelineArtifactName }}
 
-- ${{ each matrix in parameters.BuildMatrix }}:
-  - ${{ each toolchain in matrix.Toolchains }}:
-    - template: /eng/pipelines/templates/jobs/test.yml
-      parameters:
-        ServiceDirectory: ${{ parameters.ServiceDirectory }}
-        Toolchain: ${{ toolchain }}
-        Matrix: ${{ matrix }}
-        TimeoutInMinutes: ${{ parameters.TestTimeoutInMinutes }}
-
 - template: /eng/pipelines/templates/jobs/analyze.yml
   parameters:
     ServiceDirectory: ${{ parameters.ServiceDirectory }}
-    Toolchain: ${{ parameters.AnalyzeToolchain }}
+    Toolchain: stable
     TimeoutInMinutes: ${{ parameters.TestTimeoutInMinutes }}
+
+- template: /eng/common/pipelines/templates/jobs/generate-job-matrix.yml
+  parameters:
+    JobTemplatePath: /eng/pipelines/templates/jobs/ci.tests.yml
+    MatrixConfigs: ${{ parameters.MatrixConfigs }}
+    MatrixFilters: ${{ parameters.MatrixFilters }}
+    MatrixReplace: ${{ parameters.MatrixReplace }}
+    CloudConfig:
+      Cloud: public
+    SparseCheckoutPaths:
+      - /*
+      - '!/sdk/*/**/SessionRecords/*'
```

Comment:
1. This is .NET's pattern, not ours.
2. I don't think it's needed even for .NET since session records moved out into another repo and are put elsewhere. In our repo (and we copied from .NET et. al.) it's gitignore'd `.assets/` directory off the root.

Reply:
While I don't think we even need it, FWIW, our records "virtually" get put in `sdk/**/tests/data/**`.

Reply:
SessionRecords line removed

Reply:
> 1. This is .NET's pattern, not ours.
> 2. I don't think it's needed even for .NET since session records moved out into another repo and are put elsewhere. In our repo (and we copied from .NET et. al.) it's gitignore'd `.assets/` directory off the root.

@heaths unfortunately net still has some stragglers that need to clean up their recordings.